files/default/ec2_dev_2_volid.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/opt/parallelcluster/virtualenv/bin/python
```

Comment:
Is it possible to use a variable instead of a static full path here?

Reply:
ok, now uses variable